plugins/smb/komand_smb/actions/delete_file/action.py

```diff
@@ -1,42 +1,67 @@
-import komand
+import insightconnect_plugin_runtime
+from insightconnect_plugin_runtime.exceptions import PluginException
+
 from .schema import DeleteFileInput, DeleteFileOutput, Input, Output
 
-# Custom imports below
-import smb
+from smbprotocol.open import Open, ImpersonationLevel, CreateDisposition, CreateOptions, FilePipePrinterAccessMask
+from smbprotocol.file_info import FileAttributes
+from smbprotocol.exceptions import SMBException
 
 
-class DeleteFile(komand.Action):
+class DeleteFile(insightconnect_plugin_runtime.Action):
     def __init__(self):
         super(self.__class__, self).__init__(
             name="delete_file",
-            description="Delete file from share",
+            description="Delete a file from an SMB share",
```

Comment:
`.from schema import Compontent`

```suggestion
            description=Component.DESCRIPTION,
```